internal/app/machined/pkg/controllers/seccomp/seccomp.go

```diff
@@ -0,0 +1,169 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package seccomp
```

Comment:
idk, whether it makes sense to keep that under `k8s.`? or make something more generic, like `cri.`? it feels there won't be much under `seccomp.` :)

Reply:
`k8s` doesn't seem quite right, since maybe a system extension could use it :thinking: or some other runtime, it's definitely something machine specific. Maybe makes more sense to be under cri/files

Reply:
fixed